code/game/machinery/doors/firedoor.dm

```diff
@@ -1,5 +1,5 @@
 
-/var/const/OPEN = 1
+// /var/const/OPEN = 1
```

Comment:
Does this break anything?

Reply:
I don't think so, but BYOND counted that duplication as an error, because there is already a thing called OPEN in defines.